mopidy/backend.py

```diff
@@ -235,7 +235,7 @@ def change_track(self, track):
             logger.debug("Backend translated URI from %s to %s", track.uri, uri)
         if not uri:
             return False
-        self.audio.set_uri(uri).get()
+        self.audio.set_uri(uri, live_stream=track.live_stream).get()
```

Comment:
If we keep the hint in the model then we also avoid people needing to override change track, this is part of why I'm thinking about a `is_live(uri)` in the next comment.